docs/cache.md

```diff
@@ -0,0 +1,26 @@
+# Triage Metadata Caching
+
+The [wpt.fyi](https://wpt.fyi) dashboard has support for linking test results
+for a specific test and browser to issues or bugs (or more generically, any
+URL). The triaged data is stored in the
+[wpt-metadata](https://github.com/web-platform-tests/wpt-metadata) repository
+and is reflected back onto [wpt.fyi](https://wpt.fyi).
+
+This section explains the caching mechanisms for triage data in webapp and
+searchcache.
+
+## webapp
+Webapp hosts the following API endpoints:
+
+- `/api/metadata` returns WPT Metadata to be displayed on the bottom of wpt.fyi pages and via inline icons on the test-result table
+
+- `/api/metadata/triage` records triage information in the backend and sends PRs to the wpt-metadata repository which acts as the backing data store.
+
+Both endpoints share the same in-memory copy of the wpt-metadata repository, and this copy expires [every 10 minutes](https://github.com/web-platform-tests/wpt.fyi/blob/main/api/metadata_cache.go#L29) in Redis. The only exception is that when users triage metadata, /api/metadata/triage will force an update of the copy.
+
+- `/api/metadata/pending` retrieves pending metadata whose PRs are not merged yet in the wpt-metadata repository.
+
+This endpoint stores pending metadata to Redis, with [a 7-day TTL](https://github.com/web-platform-tests/wpt.fyi/blob/main/api/metadata_handler.go#L82). If its pending PRs are merged or closed, pending metadata will be cleaned from Redis in [searchcache](https://github.com/web-platform-tests/wpt.fyi/blob/main/api/query/cache/poll/poll.go#L105)
+
+## searchcache
+Searchcahe has a long-running polling thread that caches WPT Metadata [every 10 minutes](https://github.com/web-platform-tests/wpt.fyi/blob/main/api/query/cache/poll/poll.go#L89). When users search for triage information, the result is sometimes out-of-sync with webapp because the cache doesn't reflect what users have recently triaged.
```

Comment:
It might help to be explicit about if/how the search cache gets back in sync. Is only when the polling/caching happens every 10 minutes? Is there any way to force it back in sync?

The code link seems to go to a blank line (line 89). An example of my earlier comment about links to specific line number in the current file.

Nit: typo in "Searchcahe"

```suggestion
Searchcache has a long-running polling thread that caches WPT Metadata [every 10 minutes](https://github.com/web-platform-tests/wpt.fyi/blob/main/api/query/cache/poll/poll.go#L89). When users search for triage information, the result is sometimes out-of-sync with webapp because the cache doesn't reflect what users have recently triaged.
```

Reply:
Searchcache and webapp are out-of-sync because they have different update cycles. Webapp sometimes updates the cache on demand and searchcache's polling thread always runs on a 10 minute interval. Currently there is no for way to force them back in sync

Reply:
I will correct all the links

Reply:
Actually I will add a note on when they are in sync